src/StatisticalAnalyser.java

```diff
@@ -125,4 +125,16 @@ public double pathWeightMean(ArrayList<ArrayList<Path>> paths) {
         stats[1] = Math.sqrt(acc/(double)nPaths); // desvio padrao
         return stats;
     }
+
+    // Calculate routing distance -> all paths lengths / #paths
+		public double averageRoutingDistance(ArrayList<ArrayList<Path>> paths) {
+			double routingDistance = 0.0;
+
```

Comment:
We don't need to leave a space between a loop and a variable declaration when that variable is mainly used by the loop.